apps/admin/src/components/notice/NoticePost/NoticePost.hooks.tsx

```diff
@@ -0,0 +1,12 @@
+import { useNoticePostMutation } from '@/services/notice/mutations';
+import { PostNoticeReq } from '@/types/notice/remote';
+
+export const useNoticePostAction = (noticeData: PostNoticeReq) => {
+    const { noticePostMutate } = useNoticePostMutation(noticeData);
```

Comment:
```suggestion
    const { postNoticeMutate } = useNoticePostMutation(noticeData);
```

동사가 앞으로 빠지는게 더 좋을듯합니다

Reply:
아 맞네요 함순데

Reply:
나 진짜 검정고시 못해먹겠어